OsmAnd-java/src/main/java/net/osmand/router/TurnType.java

```diff
@@ -284,6 +284,35 @@ public static String lanesToString(int[] lns) {
         return s.toString();
 	}
 
+	public boolean isOnlyThroughActive() {
```

Comment:
method could be more generic and simpler ?
```
countTurnTypeDirections(TurnType checkActive, boolean onlyActive);
```

So check will look like
```
int cnt = countTurnTypeDirections(TurnType.C, true);
int cntAll = countTurnTypeDirections(TurnType.C, false);
if(cnt > 0 && cnt == cntAll) {
 .. 
}
```

Reply:
Method countTurnTypeDirections will consist in most same code as isOnlyThroughActive.